R/transformation.R

```diff
@@ -1,17 +1,20 @@
-#' Transforms new data to standardized normal (dimension 1) based on other data transformations
+#' Transforms new data to a standardized normal distribution
 #'
-#' Handled in this way in order to allow using the apply function over this function
-#' @param zx Vector where the first part is the Gaussian data, and last part is
-#' the data with the original transformation
-#' @param n_z How many elements of \code{zx} that belongs to the z-part (new data)
-#' @param type The quantile type used when back-transforming. 7 (default) is the default in \code{stats::quantile()}.
+#' @param zx Numeric vector. The first \code{n_z} items is the Gaussian data, and last part is
+#' the data with the original transformation.
+#' @param n_z Positive integer. Number of elements of \code{zx} that belongs to new data.
+#' @param type Integer between \code{1} and \code{9}. See \code{\link[stats]{quantile}} for more
```

Comment:
Couldn't we just pass type as a ... to quantile? I don't think any user will ever care to change this setting.

Reply:
We could just remove the argument, and use the default value? I don't think we should use `... `, since it could mess with the other arguments in `quantile` as well.

Reply:
Ok, let's remove it. It's only used internally anyway, so nobody will ever touch it.

Reply:
Added to #138 .